### DISCUSSION TITLE
Feature: Save resulting lab + configs as standalone Containerlab

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#638] [Features]
Netlab can already save resulting configs - small enhancement would be to also generate a Containerlab file referencing these files using startup-config entries

This would allow for a quickly reproducible lab, that can be deployed by users without Netlab

**Comment by ipspace** [comment]
Would this work for you? https://netsim-tools.readthedocs.io/en/latest/netlab/clab.html#netlab-clab-tarball

**Reply by jbemmel** [reply]
Wow, that was fast :)
